lib/rules/forbid-component-props.js

```diff
@@ -140,10 +169,48 @@ module.exports = {
         return false;
       }
 
+      function checkIsTagForbiddenByAllowList() {
+        return options.allowList.indexOf(tagName) === -1;
+      }
+
+      function checkIsTagForbiddenByAllowPatternList() {
+        if (options.allowPatternList.length === 0) {
+          return true;
+        }
+
+        return options.allowPatternList.every(
+          (pattern) => !minimatch(tagName, pattern)
+        );
+      }
```

Comment:
kind of same here? i'd rather checkIsTagForbiddenByAllowOptions be mutiline with some early returns.